assets/src/edit-story/components/panels/preset/karma/colorPresets.karma.js

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import { Fixture } from '../../../../karma/fixture';
+import { useInsertElement } from '../../../canvas';
+import { TEXT_ELEMENT_DEFAULT_FONT } from '../../../../app/font/defaultFonts';
+
+describe('Panel: Color Presets', () => {
+  let fixture;
+
+  const ADD_BUTTON = 'Add color preset';
+  const EDIT_BUTTON = 'Edit color presets';
+  const EXIT_EDIT_MODE = 'Exit edit mode';
+
+  const addText = async () => {
+    const insertElement = await fixture.renderHook(() => useInsertElement());
+    await fixture.act(() =>
+      insertElement('text', {
+        font: TEXT_ELEMENT_DEFAULT_FONT,
+        content: 'Hello, presets!',
+      })
+    );
+  };
+
+  beforeEach(async () => {
+    fixture = new Fixture();
+    await fixture.render();
+  });
+
+  afterEach(() => {
+    fixture.restore();
+  });
+
+  describe('CUJ: Creator can Apply or Save a Color from/to Their Preset Library: Display Panel', () => {
+    it('should display color presets panel for a text element', async () => {
+      await addText();
+      const addButton = fixture.screen.getByRole('button', {
+        name: ADD_BUTTON,
+      });
+      expect(addButton).toBeTruthy();
+    });
+  });
+
+  describe('CUJ: Creator can Apply or Save a Color from/to Their Preset Library: Manage Color Presets', () => {
+    it('should allow deleting a color preset', async () => {
+      // Add text element and a color preset.
+      await addText();
+      const addButton = fixture.screen.getByRole('button', {
+        name: ADD_BUTTON,
+      });
+      await addButton.click();
```

Comment:
If we use `await fixture.events.click(node)`, karma will also make sure that the target is actually clickable (on-screen, has a size, etc). That will make the tests more "real-world".

Also, the regular `node.click()` doesn't return a promise, so no `await` necessary anyway.